docs/charts/profile_columns.ipynb

```diff
@@ -0,0 +1,551 @@
+       "</script>"
+      ],
+      "text/plain": [
+       "alt.VConcatChart(...)"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import altair as alt\n",
+    "from splink.datasets import splink_datasets\n",
+    "from splink.duckdb.linker import DuckDBLinker\n",
+    "\n",
+    "df = splink_datasets.historical_50k\n",
+    "linker = DuckDBLinker(df)\n",
+    "\n",
+    "linker.profile_columns(\n",
+    "    [\"first_name\", \"gender\", \"birth_place\", \"dob\"]\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What the chart shows\n",
+    "\n",
+    "The `profile_columns` chart shows 3 charts for each selected column:\n",
+    "\n",
+    "- The left chart shows the distribution of all values in the column.\n",
```

Comment:
I think yes on both counts. 

Also, do you think there should be some more explanation (perhaps including a screenshot) of the information you can get from the tooltip for each of these charts? It feels like there is always useful information but not everyone pays attention to them.

Reply:
That was going to be one of my other comments on this work.

I think we should have something that outlines the information contained in each tooltip we produce:
* Tip full name
* Interpretation

Reply:
There's a question though about where this belongs. I think it should probably be closer to the bottom of each notebook, as to not overwhelm the user.

Perhaps a `Chart tooltips breakdown` section?

I think it's important to have something, irrespective of how short it is. It's not something we've really broken down elsewhere (at least to my knowledge).

Reply:
On your first point - that is essentially what I have written in #1513, right? Or is there a difference I am missing?

Reply:
Hey @ThomasHepworth I have added a section on tooltips for profile_columns. I'm not entirely convinced on where it sits in the structure of the page, so please have a look and let me know if you think it would work better elsewhere

Reply:
Yes, it looks as if it's already captured in https://github.com/moj-analytical-services/splink/issues/1513

Reply:
Thanks for adding some notes on the tooltips Ross, they're really valuable!

My only comment is that now we have the opposite problem, that there's a continuous stream of text on the page (broken up in places by images).

I don't know if it would make this better, but it might be worthwhile attempting:
* Bolding the titles, so they're more prominent
* Trialling breaking up sections with `<hr>` clauses.

I don't think it needs much, just something to catch the user's eye so they can distinguish between sections.

Reply:
So I have added a collapsible block for the tooltip stuff and a few `<hr>` clauses to break up some sections. Do you want to have a look and let me know what you think?